backend/app/usecase/changelog/changelog.go

```diff
@@ -0,0 +1,50 @@
+package changelog
+
+import (
+	"github.com/short-d/app/fw"
+	"github.com/short-d/short/app/entity"
+	"github.com/short-d/short/app/usecase/keygen"
+	"github.com/short-d/short/app/usecase/repository"
+)
+
+// ChangeLog represents a ChangeLog creator and retriever
+type ChangeLog interface {
+	CreateChange(title string, summaryMarkdown string) (entity.Change, error)
+	GetChangeLog() ([]entity.Change, error)
+}
+
+// Persist represents a ChangeLog creator and retriever
+type Persist struct {
+	keyGen        keygen.KeyGenerator
+	timer         fw.Timer
+	changeLogRepo repository.ChangeLog
+}
+
+// CreateChange persists a new change in the repository
+func (p Persist) CreateChange(title string, summaryMarkdown string) (entity.Change, error) {
```

Comment:
```suggestion
func (p Persist) CreateChange(title string, summaryMarkdown *string) (entity.Change, error) {
```
I rethinked about this. It's okay for the `summary` of a change to be empty. So, I changed the signature to pointer, allowing nil being passed in. The use case should be able to handle this case.